Live_data_update.sql

```diff
@@ -19,3 +21,85 @@ update cqc."Establishment" set "IsRegulated"=True,"LocationID"='1-5400068426' wh
+  select  *    from cqc."User" where "TribalID"  in (170258, 196840);
+  select  *    from cqc."EstablishmentAudit" where "EstablishmentFK" in (select distinct "EstablishmentID" from cqc."Establishment" where "TribalID"  in (170258, 196840));
+  select  *    from cqc."Establishment" where "TribalID"  in (170258, 196840);
+
+--- Record Deletion 
+
+-- Deletion based on Tribal ID [196840,170258]
+
+delete from cqc."WorkerQualifications" where "TribalID" in (170258, 196840);
+  delete from cqc."WorkerTraining" where "TribalID"  in (170258, 196840);
+  delete from cqc."WorkerAudit" where "WorkerFK" in (select distinct "ID" from cqc."Worker" where "TribalID"  in (170258, 196840));
+  delete from cqc."WorkerJobs" where "WorkerFK" in (select distinct "ID" from cqc."Worker" where "TribalID"  in (170258, 196840));
+  delete from cqc."Worker" where "TribalID"  in (170258, 196840);
+  delete from cqc."EstablishmentCapacity" where "EstablishmentID" in (select distinct "EstablishmentID" from cqc."Establishment" where "TribalID"  in (170258, 196840));
+  delete from cqc."EstablishmentJobs" where "EstablishmentID" in (select distinct "EstablishmentID" from cqc."Establishment" where "TribalID"  in (170258, 196840));
+  delete from cqc."EstablishmentServices" where "EstablishmentID" in (select distinct "EstablishmentID" from cqc."Establishment" where "TribalID"  in (170258, 196840));
+  delete from cqc."EstablishmentLocalAuthority" where "EstablishmentID" in (select distinct "EstablishmentID" from cqc."Establishment" where "TribalID"  in (170258, 196840));
+  delete from cqc."EstablishmentServiceUsers" where "EstablishmentID" in (select distinct "EstablishmentID" from cqc."Establishment" where "TribalID"  in (170258, 196840));
+
+  delete from cqc."Login" where "RegistrationID" in (select distinct "RegistrationID" from cqc."User" where "TribalID"  in (170258, 196840));
+  delete from cqc."UserAudit" where "UserFK" in (select distinct "RegistrationID" from cqc."User" where "TribalID"  in (170258, 196840));
+  delete from cqc."PasswdResetTracking" where "UserFK" in (select distinct "RegistrationID" from cqc."User" where "TribalID"  in (170258, 196840));
+  delete from cqc."User" where "TribalID"  in (170258, 196840);
+  delete from cqc."EstablishmentAudit" where "EstablishmentFK" in (select distinct "EstablishmentID" from cqc."Establishment" where "TribalID"  in (170258, 196840));
+  delete from cqc."Establishment" where "TribalID"  in (170258, 196840);
+
+
+--- Deletion Verification 
+
+-- Selection based on Tribal ID [196840,170258]
+
+
+select "TribalID"   from cqc."Establishment" where "NmdsID" IN ('E302906','G260465');
+
+SELECT * FROM cqc."Worker" where "TribalID" in (select "TribalID" from cqc."Establishment" where "NmdsID" IN ('E302906','G260465')) ;
+
+select  *    from cqc."WorkerQualification" where "TribalID" in (170258, 196840);
```

Comment:
You'll be expecting zero on everything here, so returning a count makes more sense.